contracts/token/ERC20.cairo

```diff
@@ -218,18 +218,24 @@ func transfer{
         syscall_ptr : felt*, 
         pedersen_ptr : HashBuiltin*,
         range_check_ptr
-    }(recipient: felt, amount: Uint256):
+    }(recipient: felt, amount: Uint256) -> (is_success: felt):
```

Comment:
```suggestion
    }(recipient: felt, amount: Uint256) -> (success: felt):
```
this is closer to the standard definition

> Callers MUST handle `false` from `returns (bool success)`. Callers MUST NOT assume that `false` is never returned!

Reply:
Okay, I see. The tx's will throw an error as-is, but we can make it work with an if/else

Reply:
oh no I just wanted to show that the standard calls the return variable `success`